parcels/particle.py

```diff
@@ -180,6 +181,7 @@ def __init__(self, size, grid, pclass=JITParticle,
         self.particles = np.empty(size, dtype=pclass)
         self.ptype = ParticleType(pclass)
         self.kernel = None
+        self.time_origin = grid.U.time_origin
```

Comment:
Since we need `time_origin` in the `__init__` of `ParticleFile` (see below), and that `__init__` function doesn't have access to the `field` class, I copy the value here to the `Particle` Class.
It was either that, or passing a `grid` or `field` argument to  `ParticleFile`